Add-ons/UmbracoForms/Developer/Themes/index.md

```diff
@@ -15,7 +15,7 @@ To create a theme, you need to create a folder at `/Views/Partials/Forms/Themes/
 
 Copy the explicit files you wish to override in your theme, it may be a single file or all files from the default theme folder. Make the necessary changes you desire to CSS class names, markup etc.
 
-For Umbraco 9 and previous, it's straightforward to simply copy the files you need from the default theme folder.  Umbraco 10 distributes these files as part of a Razor class library, so you won't find them on disk. The easiest way to obtain copies of the files included in the default theme is to [download this zip file](https://our.umbraco.com/FileDownload?id=23911) and extract the ones you need.
+For Umbraco 9 and previous, it's straightforward to simply copy the files you need from the default theme folder.  Umbraco 10 distributes these files as part of a Razor class library, so you won't find them on disk. The easiest way to obtain copies of the files included in the default theme is to [download this zip file](https://our.umbraco.com/FileDownload?id=24022) and extract the ones you need.
```

Comment:
Thanks - we forgot to update this link clearly after the last release, and this is the latest one at the time or writing.

Reply:
This may mean more work for you each time a new version is released, but is it worth having this link in a TinyURL type service? I seem to recall (unless I dreamt it) that Umbraco has its own similar service which would be an idea candidate for this? What would then get used for the documentation instead is the static URL, but when a version is released the URL is simply updated with the new target.

Reply:
Yes, it sounds like a good idea, and I have found that we do have such a service.  Will see if I can get access to it and set it up for this link.

Reply:
I've set one up, so if we amend this to `https://umbra.co/umbraco-forms-default-theme` that will point to the latest download.  And I can update it if and when we have an updated link.